WORKSPACE

```diff
@@ -51,9 +50,7 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 
 go_rules_dependencies()
 
-go_register_toolchains(
-    go_version = "1.19.1",
-)
+go_register_toolchains(version = "1.20.5")
```

Comment:
Any reason for going for 1.20.5 over some of the later releases?

Reply:
That's the one given in the [bazel go rules release](https://github.com/bazelbuild/rules_go/releases/tag/v0.41.0) and that's the one also referenced in [bazel gazelle](https://github.com/bazelbuild/bazel-gazelle#running-gazelle-with-bazel). I tried latest v0.42, but that didn't seem to work with gazelle.

Reply:
I guess gazelle is not ready for 1.21 yet :shrug:

Reply:
Ok thanks Daniel